Formula/qt5.rb

```diff
@@ -73,6 +73,7 @@ class Qt5 < Formula
   option "with-oci", "Build with Oracle OCI plugin"
   option "with-qtwebkit", "Build with QtWebkit module"
   option "without-webengine", "Build without QtWebEngine module"
+  option "with-openssl", "Build with openssl"
```

Comment:
`=> :optional` and `=> :recommended` generate option descriptions automatically. Since this description doesn't add any additional information, you can just delete the line entirely.